<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List</class>
 <widget class="QWidget" name="List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>118</width>
    <height>324</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="labelUnsaved">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>3</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>3</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QListWidget" name="list">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="TickerMiniInput" name="widgetInput" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="InlineTextInput" name="widgetSearch" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetButtons" native="true">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="pushList">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>24</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string extracomment="Noun">List</string>
        </property>
        <property name="icon">
         <iconset resource="tht.qrc">
          <normaloff>:/images/list.png</normaloff>:/images/list.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="pushAdd">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>24</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="icon">
         <iconset resource="tht.qrc">
          <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushSave">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>24</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string extracomment="This is the label on a button that user pushes to issue the command. Ticker is a short company name, see http://www.investopedia.com/terms/t/tickersymbol.asp . The common practice is to borrow &quot;ticker&quot; from English and incorporate into your language (see http://en.wikipedia.org/wiki/Loanword)">Save tickers</string>
        </property>
        <property name="icon">
         <iconset resource="tht.qrc">
          <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="pushSaveAs">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>24</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="icon">
         <iconset resource="tht.qrc">
          <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>3</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widgetHeader" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackHeader">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_3">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelHeader">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="InlineTextInput" name="widgetEnterHeader" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TickerMiniInput</class>
   <extends>QWidget</extends>
   <header>tickerminiinput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InlineTextInput</class>
   <extends>QWidget</extends>
   <header>inlinetextinput.h</header>
   <container>1</container>
   <slots>
    <signal>cancel()</signal>
    <signal>next()</signal>
    <signal>textChanged(QString)</signal>
    <signal>accept()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushList</tabstop>
  <tabstop>pushAdd</tabstop>
  <tabstop>pushSave</tabstop>
  <tabstop>pushSaveAs</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources>
  <include location="tht.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushSave</sender>
   <signal>clicked()</signal>
   <receiver>List</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>List</receiver>
   <slot>loadItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetSearch</sender>
   <signal>cancel()</signal>
   <receiver>List</receiver>
   <slot>stopSearching()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetSearch</sender>
   <signal>next()</signal>
   <receiver>List</receiver>
   <slot>slotSearchTickerNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetSearch</sender>
   <signal>textChanged(QString)</signal>
   <receiver>List</receiver>
   <slot>slotSearchTicker(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>List</receiver>
   <slot>slotCurrentRowChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetSearch</sender>
   <signal>accept()</signal>
   <receiver>List</receiver>
   <slot>stopSearching()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetEnterHeader</sender>
   <signal>cancel()</signal>
   <receiver>List</receiver>
   <slot>headerCancelled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetEnterHeader</sender>
   <signal>accept()</signal>
   <receiver>List</receiver>
   <slot>headerAccepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSave()</slot>
  <slot>loadItem()</slot>
  <slot>stopSearching()</slot>
  <slot>slotSearchTickerNext()</slot>
  <slot>slotSearchTicker(QString)</slot>
  <slot>slotCurrentRowChanged(int)</slot>
  <slot>headerCancelled()</slot>
  <slot>headerAccepted()</slot>
 </slots>
</ui>
